src/main.cpp

```diff
@@ -640,13 +731,17 @@ void bleLoop(){
 }
 
 void resetLoop(){
-  if (wifiOn){    
-        if (resetvar == 1199) {      
-        resetvar = 0;
-        delay(45000);   // 45 seconds, reset at 30 seconds
+  if (wifiOn){
```

Comment:
Please resolve this issue first in previous PR.

Reply:
commented in previous PR